src/pages/home/report/ReportActionCompose.js

```diff
@@ -57,21 +61,32 @@ const defaultProps = {
 class ReportActionCompose extends React.Component {
     constructor(props) {
         super(props);
+
         this.updateComment = this.updateComment.bind(this);
         this.debouncedSaveReportComment = _.debounce(this.debouncedSaveReportComment.bind(this), 1000, false);
         this.debouncedBroadcastUserIsTyping = _.debounce(this.debouncedBroadcastUserIsTyping.bind(this), 100, true);
         this.submitForm = this.submitForm.bind(this);
         this.triggerSubmitShortcut = this.triggerSubmitShortcut.bind(this);
         this.submitForm = this.submitForm.bind(this);
         this.setIsFocused = this.setIsFocused.bind(this);
+        this.showEmojiPicker = this.showEmojiPicker.bind(this);
+        this.hideEmojiPicker = this.hideEmojiPicker.bind(this);
+        this.addEmojiToTextBox = this.addEmojiToTextBox.bind(this);
         this.focus = this.focus.bind(this);
         this.comment = props.comment;
 
         this.state = {
             isFocused: true,
             textInputShouldClear: false,
             isCommentEmpty: props.comment.length === 0,
+            isEmojiPickerVisible: false,
             isMenuVisible: false,
+
+            // The horizontal and vertical position (relative to the screen) where the emoji popover will display.
```

Comment:
Not trying to split hairs, but I think it's relative to the window, not the screen 😉 

```suggestion
            // The horizontal and vertical position (relative to the window) where the emoji popover will display.
```